core/bin/linfo

```diff
@@ -116,7 +119,13 @@ parseCmdLine() {
       esac
       shift
    done
-   
+
+   CONF_LIST="$1"
+   while [ ${#} -gt 0 ]
+   do
+      CONF_LIST="$CONF_LIST $2"
+      shift
+   done   
    if [ $# -gt 0 ]; then
```

Comment:
This `if` statement becomes redundant now that the command allows non-option arguments